application-templates/starter/src/test-utils/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  renderApp,
+  renderAppWithRedux,
+} from '@commercetools-frontend/application-shell/test-utils';
+import { AsyncApplicationRoutes } from '../components/entry-point';
```

Comment:
@emmenko follow-up: we shouldn't render async components here I think. We had issues with that internally.